homeassistant/components/hive/__init__.py

```diff
@@ -15,8 +15,7 @@
 from homeassistant.const import CONF_SCAN_INTERVAL
 from homeassistant.core import HomeAssistant
 from homeassistant.exceptions import ConfigEntryAuthFailed, ConfigEntryNotReady
-from homeassistant.helpers import aiohttp_client
-from homeassistant.helpers.device_registry import DeviceEntry
+from homeassistant.helpers import aiohttp_client, device_registry as dr
```

Comment:
I would suggest to move the (dependency bump + hassfest fix) part to a preliminary PR
And keep this PR for just the device registry fix.